environment/src/test/java/com/sequenceiq/environment/environment/service/LoadBalancerPollerServiceTest.java

```diff
@@ -65,33 +66,35 @@ public class LoadBalancerPollerServiceTest {
 
     private final FlowEndpoint flowEndpoint = Mockito.mock(FlowEndpoint.class);
 
+    private final EntitlementService entitlementService = Mockito.mock(EntitlementService.class);
+
     private final RegionAwareInternalCrnGeneratorFactory regionAwareInternalCrnGeneratorFactory
             = Mockito.mock(RegionAwareInternalCrnGeneratorFactory.class);
 
     private final RegionAwareInternalCrnGenerator regionAwareInternalCrnGenerator
             = Mockito.mock(RegionAwareInternalCrnGenerator.class);
 
     private final LoadBalancerPollerService underTest =
-        new LoadBalancerPollerService(datahubService, sdxService, stackService, flowEndpoint, regionAwareInternalCrnGeneratorFactory);
+        new LoadBalancerPollerService(datahubService, sdxService, stackService, flowEndpoint, regionAwareInternalCrnGeneratorFactory, entitlementService);
```

Comment:
Please remove the explicit constructor invocation and add the `@InjectMocks` annotation instead.